### PR TITLE
TB Management - Case management, TB Suspect adjustments

### PR DESCRIPTION
**Pull request by anvmn**
#1052

### CODE REVIEW

File: client/src/elm/Pages/GlobalCaseManagement/Utils.elm [hunk 1]
```diff
@@ -232,13 +237,67 @@ generateImmunizationFollowUps limitDate followUps =
...
+One is Acute illness follow up, in case Tuberculosis suspect is diagnosed.
+Other is the follow ups from Tuberculosis management encounter.
+So, in order to combine these 2, we'll have to include AI follows for person,
+as part of Tuberculosis follow ups.
+Currently, all Tuberculosis encounters are derived from single participant (as
+there's no option to 'end' Tuberculosis illness), so this solution is valid.
+In case we'll have multiple Tuberculosis illnesses in future, this wil need to
+be revised.
+There's also an option that patient did not attend Tuberculosis encounter, and
+there's no participant. This is why we return second dictionary derrived from
```
Comment (AronNovak) [inline client/src/elm/Pages/GlobalCaseManagement/Utils.elm]:
```suggestion
there's no participant. This is why we return second dictionary derived from
```

File: server/hedley/modules/custom/hedley_case_management/hedley_case_management.module [hunk 2]
```diff
@@ -361,14 +368,17 @@ function hedley_case_management_handle_tuberculosis_insert_trigger($node) {
...
+ *
  * @param int $person_id
  *   Person ID.
  * @param int $to_nid
  *   Optional; IDs of loaded follow-ups should be below this node ID.
  * @param int $limit
  *   Maximal number of follow-ups to load. Defaults to 200.
  */
-function hedley_case_management_resolve_tuberculosis_follow_ups($person_id, $to_nid = NULL, $limit = 200) {
+function hedley_case_management_resolve_tuberculosis_follow_ups($person_id, $to_nid = NULL, $limit = 1000) {
```
Comment (AronNovak) [inline server/hedley/modules/custom/hedley_case_management/hedley_case_management.module]: Why did we need to bump the limit from 200 to 1000? Were there such cases with more than 200 follow-ups? It sounds suspicious at first sight.

Comment (anvmn) [reply server/hedley/modules/custom/hedley_case_management/hedley_case_management.module]: Well, no. Person should not have more than 200 follow  ups, specially, that follow ups are getting resolved, and excluded from query results. So I'll revert the change.

Comment (anvmn) [inline server/hedley/modules/custom/hedley_case_management/hedley_case_management.module]:
```suggestion
function hedley_case_management_resolve_tuberculosis_follow_ups($person_id, $to_nid = NULL, $limit = 200) {
```

**Review by AronNovak — APPROVED** [review]

**Pull request merged**